freqtrade/optimize/hyperopt.py

```diff
@@ -259,13 +262,16 @@ def generate_optimizer(self, _params: Dict) -> Dict:
         """
         params = self.get_args(_params)
         if self.has_space('roi'):
-            self.backtesting.strategy.minimal_roi = self.custom_hyperopt.generate_roi_table(params)
+            self.backtesting.strategy.minimal_roi = \
+                    self.custom_hyperopt.generate_roi_table(params)
```

Comment:
is this new linewrap cosmetic (because it looks better) or for flake8?

just wondering since i don't see anything else changed and i wonder if i missed something...

Reply:
this one is to align to couple of next lines, that do not fit into the limit